mayday.go

```diff
@@ -58,8 +68,15 @@ func main() {
 		log.Fatal(err)
 	}
 
+	journals, err := mayday.GetJournals()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var tarables []mayday.Tarable
+
 	for i := 0; i < len(files); i++ {
```

Comment:
As far as I can tell you're not passing any references in the body of this loop, so reuse should be a non-issue?